internal/pkg/mapper/description/description.go

```diff
@@ -0,0 +1,32 @@
+package description
+
+import (
+	"strings"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/model"
+)
+
+// descriptionMapper normalize object description loaded from API.
+// Description is normalized in the same way as when reading a local "description.md" file.
+// The white characters at the end are removed.
+type descriptionMapper struct{}
+
+func NewMapper() *descriptionMapper {
+	return &descriptionMapper{}
+}
+
+func (m *descriptionMapper) MapAfterRemoteLoad(recipe *model.RemoteLoadRecipe) error {
+	switch o := recipe.InternalObject.(type) {
+	case *model.Branch:
+		o.Description = m.normalize(o.Description)
```

Comment:
- pri ukladani `description.md` odstranujeme biele znaky na konci:
https://github.com/keboola/keboola-as-code/blob/e59b45c6d063dcaa99ea73b5f42d29fe77e87073/internal/pkg/local/save.go#L68-L71
- podobne aj pri nacitavani `description.md`:
- https://github.com/keboola/keboola-as-code/blob/e59b45c6d063dcaa99ea73b5f42d29fe77e87073/internal/pkg/filesystem/aferofs/filesystem.go#L415
- robi sa to, pretoze bezne IDE prida na koniec suboru novy riadok
- a description moze byt v API bez "\n", iba jedena veta
- no chybalo to pri nacitani z API
- co sa prejavilo tak, ze diff ukazoval rozdiel, ak v API koncila `description` bielym znakom